server/rest/src/main/java/org/infinispan/rest/resources/SecurityResource.java

```diff
@@ -47,14 +55,17 @@
    private final String accessGrantedPath;
    private final String accessDeniedPath;
```

Comment:
unused

Reply:
In theory that is something we will use for the console, when authentication fails